cli/cmd/project/jwt.go

```diff
@@ -36,6 +36,9 @@ func JwtCmd(ch *cmdutil.Helper) *cobra.Command {
 				if err != nil {
 					return err
 				}
+			} else if cmd.Flags().Changed("project") {
+				// Use the flag value directly when provided
+				name = cmd.Flag("project").Value.String()
```

Comment:
Something's off here – this shouldn't be needed. When Cobra parses flags, it automatically assigns values into the pointer passed in `jwtCmd.Flags().StringVar(&name, "project", "", "Project Name")`.